### PR TITLE
Update .gitattributes

### PR DESCRIPTION
**Pull request by Evangelink**

### CODE REVIEW

File: .gitattributes [hunk 1]
```diff
@@ -1,63 +1,54 @@
...
-# image files are treated as binary by default.
-###############################################################################
-#*.jpg   binary
-#*.png   binary
-#*.gif   binary
+# scripts
+*.sh        text eol=lf
+*.bat       text eol=crlf
+*.cmd       text eol=crlf
+*.ps1       text eol=crlf
```
Comment (michalb-sonar) [inline .gitattributes]: LF works on Windows, so that should probably be LF.

File: .gitattributes [hunk 2]
```diff
@@ -1,63 +1,54 @@
...
+*.bmp       binary
+*.gif       binary
+*.jpg       binary
+*.png       binary
+*.class     binary
+*.dll       binary
+*.ico       binary
+*.pfx       binary
+*.snk       binary
+.exe        binary
```
Comment (michalb-sonar) [inline .gitattributes]: *.exe

File: .gitattributes [hunk 3]
```diff
@@ -1,63 +1,54 @@
...
-#*.dot   diff=astextplain
-#*.DOT   diff=astextplain
-#*.pdf   diff=astextplain
-#*.PDF   diff=astextplain
-#*.rtf   diff=astextplain
-#*.RTF   diff=astextplain
+# documentation
+*.md        text
+*.tt        text
+.markdown   text
```
Comment (michalb-sonar) [inline .gitattributes]: *.markdown?

File: .gitattributes (hunk 1 above)
Comment (Evangelink) [reply .gitattributes]: There were comments saying there are some bugs with LF so for now I will keep it as CRLF.

File: .gitattributes (hunk 3 above)
Comment (Evangelink) [reply .gitattributes]: removed

File: .gitattributes (hunk 1 above)
Comment (michalb-sonar) [reply .gitattributes]: Good to know. Do you have a link?

**Pull request merged**